prep.sh

```diff
@@ -32,7 +32,7 @@ for LINE in $LINEZ; do
 	echo '# nothing' >jenkins/split-plugins.txt
 	cp -r jenkins-for-test "megawar-${LINE}"
 	jar uvf megawar-$LINE/WEB-INF/lib/jenkins-core-*.jar jenkins/split-plugins.txt
-	rm -rfv `# TODO delete all but instance-identity? megawar-$LINE/WEB-INF/detached-plugins` megawar-$LINE/META-INF/*.{RSA,SF}
```

Comment:
No longer needed as of https://github.com/jenkinsci/maven-hpi-plugin/pull/336.

Reply:
Hopefully? I failed to code/test this properly the first time around, fixed in ace1efa138b9b796771b8fb35858ecc53341056d but not yet tested. If the tests fail I will just revert this hunk.

Reply:
[Worked like a champ](https://ci.jenkins.io/job/Tools/job/bom/job/PR-1336/2/) 💪

Reply:
(FTR, reverts a workaround added in #1160)

Reply:
That is a good sign!